### PR TITLE
Refactor for lesions analysis - Bundles

### PR DESCRIPTION
**Pull request by frheault**
Added a new channels for lesions, a new lesions process and its two aggregate counterpart (whole bundle vs per_point).

This does not change a thing if you do not provide a lesion_mask.nii.gz
If you do provide it, it will add a few new metrics to json/xlsx/png similar to before.

To keep it simple, the structure of json was keep as close to existing json in the pipeline.
However this make the json/plot a bit weird. For example, for metrics of a specific subject you have mean/std per point. Which make sense because you have thousand of streamlines and so you have a lot to average.
AF_L_fa_metric

For lesions, you have a similar metrics: lesion_volume, which is avg/std of the volume of each lesion
image

But you also have lesion count and total volume, which by design do not have std. However, for simplicity of code everywhere and pure hell in convert to xlsx as well as merging json. The std of 0 will be saved for these two metrics.
AF_L_lesion_count
AF_L_lesion_total_volume

(At the population level, total volume and lesions count have std, since it comes for multiple subjects)
This is not perfect, but I think this would have been enough to speed up Ann-Marie processing phase of her project by a lot.

PS: If you have 100 regions and almost no lesions, this visualisation makes little to no sense. But the data is available in json/xlsx to make custom graph that will look much better, and the analysis can be done in SPSS like AM did.

PPS: There is also lesions-specific information in the folder subj/Lesion_Load/*__lesions_stats.json. This is not averaged at the population level since everyone has different, unmatched lesions.

### CODE REVIEW

File: USAGE [hunk 1]
```diff
@@ -13,6 +13,7 @@ DESCRIPTION
 
                             [root]
                             ├── S1
+                            │   ├── *lesion_mask.nii.gz (optional)
```
Comment (mdesco) [inline USAGE:16]: Please add *lesion_mask.nii.gz (optional) in S2 as well below

Comment (mdesco) [reply USAGE:16]: Also, can you please add 1-2 sentences in the first USAGE text "Compute tractometry on each bundles/metrics" to describe the optional lesion_mask option. What does it do, what does it change in the outputs. Thanks

**Review by mdesco — COMMENTED** [review]
Just tested and it works well! I really like it. So, after the minor edits, this is ready to go, in conjunction with PR/371

File: USAGE (hunk 1 above)
Comment (mdesco) [reply USAGE:16]: Also, should the --min_lesion_vol be given as an option to the user somewhere?

Comment (frheault) [reply USAGE:16]: @mdesco It is now a parameter

**Pull request merged**